src/data/directory/lgu/region-i-ilocos-region.json

```
@@ -570,17 +567,17 @@
             "name": "MARIO B. SUBAGAN"
           },
           "vice_mayor": {
-            "name": "SAMUEL B. SUBAGAN JR."
+            "name": "RENATO NGANGAC"
           }
         },
         {
           "municipality": "Tagudin",
           "zip_code": "2714",
           "mayor": {
-            "name": "ROQUE S. VERZOSA JR."
+            "name": "EVANGELINE I. VERZOSA"
           },
           "vice_mayor": {
-            "name": "EVANGELINE I. VERZOSA"
+            "name": "BERNARDO TOVERA JR"
```

```suggestion
            "name": "BERNARDO F. TOVERA JR."
```

ref: https://lgu201.dilg.gov.ph/view.php?r=01&p=029&m=33